src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1379,6 +1502,57 @@ internal static IEnumerable<string> FindIdentifiersForSelectedNodes(IEnumerable<
             return selectedNodes.SelectMany(n => n.OutPorts.Select(p => n.GetAstIdentifierForOutputIndex(p.Index).Value));
         }
 
+        /// <summary>
+        /// Find all output identifiers for custom nodes in the provided workspace. 
+        /// </summary>
+        /// <param name="workspace">A workspace</param>
+        /// <returns></returns>
+        internal static IEnumerable<string> FindIdentifiersForCustomNodes(HomeWorkspaceModel workspace)
+        {
+            if (workspace == null)
+            {
+                return null;
+            }
+
+            // Remove the output identifier appended to the custom node outputs.
+            const string pattern = "_out[0-9]";
+            var rgx = new Regex(pattern);
```

Comment:
If you just remove the `pattern` variable and place it as an argument it will just get burned into the callsite and be cleaner.